firefox-ios/Client/Ecosia/UI/NTP/MultiPurposeHeader/NTPMultiPurposeEcosiaHeaderViewModel.swift

```diff
@@ -88,8 +75,7 @@ extension NTPMultiPurposeEcosiaHeaderViewModel: HomepageViewModelProtocol, Featu
     }
 
     func setTheme(theme: Theme) {
-        // Ecosia: Use applyTheme for consistency (following FeedbackView pattern)
-        applyTheme(theme: theme)
+        self.theme = theme
```

Comment:
I guess `theme` should be also moved out of the ViewModel and as a state of the View? Not sure if the header is a specific case...